packages/admin-ui/src/Label/Label.tsx

```diff
@@ -10,12 +9,21 @@ const labelVariants = cva(
     "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
 );
 
-const Label = React.forwardRef<
-    React.ElementRef<typeof LabelPrimitive.Root>,
-    React.ComponentPropsWithoutRef<typeof LabelPrimitive.Root> & VariantProps<typeof labelVariants>
->(({ className, ...props }, ref) => (
-    <LabelPrimitive.Root ref={ref} className={cn(labelVariants(), className)} {...props} />
-));
-Label.displayName = LabelPrimitive.Root.displayName;
+interface LabelProps
+    extends Omit<React.ComponentPropsWithoutRef<typeof LabelPrimitive.Root>, "children">,
+        VariantProps<typeof labelVariants> {
+    text: React.ReactNode;
+}
+
+const LabelBase = React.forwardRef<React.ElementRef<typeof LabelPrimitive.Root>, LabelProps>(
+    ({ className, text, ...props }, ref) => (
+        <LabelPrimitive.Root ref={ref} className={cn(labelVariants(), className)} {...props}>
+            {text}
```

Comment:
No, it needs to be a `<label>` element that labels a form control. I would keep "dummy" `Text` component separated from `Label` because: 

- they have different semantic meaning;
- in our Figma file, labels have some distinctive properties